docker-compose-files/dspace-compose/README.md

```diff
@@ -116,20 +102,23 @@ winpty docker exec -it --detach-keys "ctrl-p" dspacedb psql -U dspace
 - DSpace 7: [http://localhost:8080/spring-rest](http://localhost:8080/spring-rest)
 
 ## 6. Stopping DSpace
-To stop DSpace, the following command can be run.  The image will be retained in a stopped state.
+To stop DSpace, the following command can be run.  The containers will be retained in a stopped state.
+Any code or configuration changes that you made to your containers will be retained.
 ```
-docker-compose -p $DPROJ stop
+docker-compose -p d6 stop
 ```
 
-You can destroy the images with the following command.  This command will be necessary to run if you change the compose file that you are using.
+You can destroy the images with the following command.  Any code changes that you have made within the containers will be lost.
+The code directories will be refreshed from the docker images on recreation.  
+Note that the contents of the assetstore, database, and solr repositories will be persisted as docker volumes.
 
 ```
-docker-compose -p $DPROJ down
+docker-compose -p d6 down
```

Comment:
For the `d7` image, I had to run `docker-compose -p d7 down --remove-orphans` because of these: 

> WARNING: Found orphan containers (dspace-angular) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.

> ERROR: error while removing network: network d7_dspacenet id 8aa8253fe03b39a7f8b38693f635635c59b0e9d260d38e2b077949251512c2aa has active endpoints

Reply:
Documentation updated